src/test/java/com/glovoapp/ownership/ParentPackageAnnotationScannerTest.java

```diff
@@ -0,0 +1,59 @@
+package com.glovoapp.ownership;
+
+import com.glovoapp.ownership.examples.ExampleOwnershipAnnotation;
+import com.glovoapp.ownership.examples.parentpackage.childpackage.NotAnnotatedClass;
+import com.glovoapp.ownership.notannotatedpackage.childpackage.ClassInNotAnnotatedPackage;
+import lombok.var;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ParentPackageAnnotationScannerTest {
+
+    @Test
+    void getPackageName_givenClass_shouldReturnPackageName() {
+
+        var sut = new ParentPackageAnnotationScanner(ExampleOwnershipAnnotation.class);
```

Comment:
Fun fact: `sut` means `nipple` in Polish.

```suggestion
        var scanner = new ParentPackageAnnotationScanner(ExampleOwnershipAnnotation.class);
```

Reply:
![image](https://user-images.githubusercontent.com/53642942/140277305-3538b1d6-4e3a-4123-b040-594d353bb1a0.png)